packages/UI/src/context/UserDataContext.js

```diff
@@ -53,8 +53,8 @@ function useUserDataHook() {
         });
       })
       .catch((e) => {
-        console.error('failed to fetch', e);
-        setIsAuthenticated(false);
+        console.error('Failed to fetch', e);
+        setIsAuthenticated(true);
```

Comment:
Why do we set authenticated to true when there is an error?

Reply:
Not sure, I did not add this.